src/interface/drawstatus.cpp

```diff
@@ -2091,18 +2060,19 @@ void drawStatus(int player)
 				hotbar_t.hotbarTooltipLastGameTick = std::max(ticks - TICKS_PER_SECOND, ticks - hotbar_t.hotbarTooltipLastGameTick);
 
 				//Activate a hotbar slot if in-game.
-				*inputPressedForPlayer(player, impulses[IN_HOTBAR_SCROLL_SELECT]) = 0;
-				inputs.controllerClearInput(player, INJOY_GAME_HOTBAR_ACTIVATE);
 				item = uidToItem(hotbar[hotbar_t.current_hotbar].item);
 			}
 
-			if ( !shootmode && inputs.bControllerInputPressed(player, INJOY_MENU_HOTBAR_CLEAR) && !players[player]->bookGUI.bBookOpen ) //TODO: Don't activate if any of the previous if statement's conditions are true?
+            // We don't have a hotbar clear binding, but if we need one, feel free to add it
```

Comment:
I think we'll deprecate this, too many accidental hotbar clearings. But will make a note to revisit